compose-nvidia.yml

```diff
@@ -0,0 +1,36 @@
+# ./sail -f compose-nvidia.yml down && ./sail build && ./sail -f compose-nvidia.yml build && ./sail -f compose-nvidia.yml up -d && ./sail shell
```

Comment:
this supposed to be here?

Reply:
has been removed and replaced with a more convenient solution